sdk/storage/src/core/errors.rs

```diff
@@ -66,3 +68,9 @@ impl From<azure_core::errors::ParsingError> for AzureStorageError {
         AzureStorageError::ParsingError(err)
     }
 }
+
+impl From<azure_core::errors::ParseError> for AzureStorageError {
+    fn from(err: azure_core::errors::ParseError) -> AzureStorageError {
```

Comment:
You can remove this is you just add `#[from]` above.

Reply:
Nice! Changed 👍